requirements/base.txt

```diff
@@ -1,7 +1,7 @@
-click>=2.0, <8.0
-gitpython~=2.1
-invoke==0.11.1
-semver~=2.2
+click>=7.0, <8.0
```

Comment:
We have tried to update click before but having this tight a range on click causes problems for users and we do not use any versions specific to seven so please put this back?

Reply:
It should be covered on the test environment, as it should work on python 3.4 and higher.

I don't have any problem lowering `click` to 2.0 as minimum required version but should be good to know which issues are caused by bumping this dependency.

Reply:
See #155 for more info

Reply:
I see, so far all test are passing with `click>= 7.0`. I will check again the version and share the test results.

Reply:
Okay, let's leave it and see how it goes.